app/src/main/res/layout/activity_main.xml

```diff
@@ -19,6 +19,7 @@
                 android:id="@+id/srwMovieList"
                 android:layout_width="match_parent"
                 android:layout_height="match_parent"
+                android:background="@color/background"
```

Comment:
You could use a more specific name to describe this background.

Reply:
Got it!